docs/.sphinx/.markdownlint.json

```diff
@@ -0,0 +1,16 @@
+{
+    "default": false,
+    "MD003": { "style": "atx" },
+    "MD013": { "code_blocks": false, "tables": false, "stern": true, "line_length": 150},
+    "MD014": true,
+    "MD018": true,
+    "MD022": true,
+    "MD023": true,
+    "MD026": { "punctuation": ".,;。，；"},
+    "MD031": { "list_items": false},
+    "MD032": true,
+    "MD035": true,
+    "MD042": true,
+    "MD045": true,
+    "MD052": true
+}
```

Comment:
A few of the files are missing final new lines 🙂

Reply:
Sorry, I don't understand what you mean? Are you saying all the files should end with an empty new line, and if that's the case, then why?

Reply:
Perhaps don't worry about it too much. It is a bit arbitrary, most files in a code repo should have a final new line. I think JSON files actually don't usually. It is easier for people to add more content